src/challenges.js

```diff
@@ -1,26 +1,44 @@
 /* ======================== CallBacks Practice ============================ */
-const each = (elements, cb) => {
-  // Iterates over a list of elements, yielding each in turn to the `cb` function.
-  // This only needs to work with arrays.
-};
+// let myArr = [1, 2, 3, 4]
 
-const map = (elements, cb) => {
-  // Produces a new array of values by mapping each value in list through a transformation function.
-  // Return the new array.
+const each = (elements, cb) => { // the forEach function! 
+  for (let i = 0; i < elements.length; i++){// Iterates over a list of elements, yielding each in turn to the `cb` function.
+  cb(elements[i], i);
+}// This only needs to work with arrays.
```

Comment:
This looks like it's working and you just need things to be formatted properly! You can do `shift + alt + f` to format .js files!